### ISSUE TITLE
Datemenu broken in 5.3 and TYPO3 7.6

### ISSUE THREAD

**Issue by georgringer** [georgringer/news#168]

**Comment by schugabe** [comment]
git bisect yields this commit as reason

```
93aa78317569d0b4e31e145c3a474d8c1621290e is the first bad commit
commit 93aa78317569d0b4e31e145c3a474d8c1621290e
Author: Martin Hummer <ma.hummer@gmail.com>
Date:   Fri Oct 28 10:20:31 2016 +0200

[BUGFIX] Fix raw SQL handling for 8.4
    
Resolves: https://github.com/georgringer/news/pull/139/
```

**Comment by martinhummer** [comment]
should be resolved by:
https://github.com/georgringer/news/pull/170
